### ISSUE TITLE
✅ 🔗 🐞 Bug Report: Menu Widget missing 'None' responsive breakpoint (Duplicate of #21796)

### ISSUE THREAD

**Issue by obolzer** [elementor/elementor#23225]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The Menu widget (Mega Menu) lacks the mobile breakpoint 'None' for its responsive settings. The Wordpress Menu (Nav Menu) has it.

Background: I want to create a Mega Menu that is kept showing even on mobile. It includes a some fixed text menus and a dropdown menu without text but only an (hamburger) icon toggling a complex Container. Below the mobile breakpoint, the Menu is hidden behind a hamburger icon,  requiring two interactions to show the Container, which is not the intended outcome.

I hope it is a simple omission and can be added in a future release.
Looks like all that is missing is the 
$dropdown_options['none'] = esc_html__( 'None', 'elementor-pro' );
line before the add_control and a bit of corresponding CSS.

### Steps to reproduce

1) Select placed Menu Widget
2) Contents -> Responsive Settings -> Breakpoint
3) Only available options: Mobile Portrait (<767px), Tablet Portrait (>1024px).  Missing 'None'

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


					
== Server Environment ==
	Operating System: FreeBSD
	Software: Apache
	MySQL version: MySQL Community Server (GPL) v5.7.40
	PHP Version: 7.4.33
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 50M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: https://wptest.skwea.co.jp
	Home URL: https://wptest.skwea.co.jp
	WP Multisite: No
	Max Upload Size: 20 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: ja
	Timezone: Asia/Tokyo
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.8.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: ja
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields
		Version: 6.1.7
		Author: WP Engine

	Advanced Post Queries
		Version: 1.1.1
		Author: Shabti Kaplan

	All In One WP Security
		Version: 5.2.1
		Author: All In One WP Security & Firewall Team

	Breeze
		Version: 2.0.27
		Author: Cloudways

	Elementor
		Version: 3.14.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.14.0
		Author: Elementor.com

	ImageMagick Engine
		Version: 1.7.7
		Author: Orangelab

	Protect Uploads
		Version: 0.5.2
		Author: alticreation

	PublishPress Permissions
		Version: 3.10.0
		Author: PublishPress

	Redirection
		Version: 5.3.10
		Author: John Godley

	Remove Page Title From Elementor Pages by Default
		Version: 1.0.1
		Author: Atdhe Boshnjaku

	Remove RSS Feed
		Version: 1.2.1
		Author: Boopathi Rajan

	SAKURA RS WP SSL
		Version: 1.4.0
		Author: SAKURA Internet Inc.

	Simply Static
		Version: 3.0.3
		Author: Patrick Posner

	Table Addons for Elementor
		Version: 1.3.0
		Author: Iqbal Bary

	TinyPNG - JPEG, PNG & WebP image compression
		Version: 3.4.1
		Author: TinyPNG

	WP Multibyte Patch
		Version: 2.9
		Author: Seisuke Kuraishi

	Yoast Duplicate Post
		Version: 4.5
		Author: Enrico Battocchi & Team Yoast


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	最適化済み DOM 出力: デフォルトで有効
	アセット読み込みの改善: デフォルトで有効
	Improved CSS Loading: デフォルトで有効
	Inline Font Icons: デフォルトで無効
	アクセシビリティの改善: デフォルトで有効
	Additional Custom Breakpoints: デフォルトで有効
	admin_menu_rearrangement: デフォルトで無効
	Flexbox Container: 有効
	Upgrade Swiper Library: デフォルトで有効
	Grid Container: デフォルトで無効
	Default to New Theme Builder: デフォルトで有効
	Hello テーマヘッダーとフッター: デフォルトで有効
	Editor Top Bar: 有効
	ランディングページ: デフォルトで有効
	Nested Elements: 有効
	Nested Accordion: デフォルトで無効
	Lazy Load Background Images: デフォルトで無効
	Global Style Guide: デフォルトで無効
	Page Transitions: デフォルトで有効
	Notes: デフォルトで有効
	Loop: デフォルトで有効
	Form Submissions: デフォルトで有効
	Scroll Snap: デフォルトで有効
	Menu: 有効


	

				
````

</details>

**Comment by nicholaszein** [comment]
Hi, @obolzer,

Thank you for reaching out! 🙏 

### ⛔ After reviewing your report we concluded this is not a bug caused by Elementor. The feature you're referring to is not yet supported.

If you think this should be an Elementor feature, please cast your ⬆️ **Upvote** in #21796 and we will consider it in the future if there is enough demand (votes) from other users.

Kind regards

**Comment by nicholaszein** [comment]
Hello there @obolzer!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.17.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
